File: app/src/androidTest/java/ru/roscha_akademii/medialib/AssertsFileHelper.java
package ru.roscha_akademii.medialib;

import android.app.Instrumentation;

import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStream;
import java.io.InputStreamReader;

public class AssertsFileHelper {
    public static String readFileFromAsserts(Instrumentation instrumentation, String file) throws IOException {
        InputStream is = instrumentation.getContext().getResources().getAssets().open(file);

        final StringBuilder sb = new StringBuilder();

        String strLine;
        BufferedReader reader = new BufferedReader(new InputStreamReader(is));

        while ((strLine = reader.readLine()) != null) {
            sb.append(strLine);
        }

        return sb.toString();
    }

}

File: app/src/androidTest/java/ru/roscha_akademii/medialib/AssertsFileHelper.kt
package ru.roscha_akademii.medialib

import android.app.Instrumentation

import java.io.BufferedReader
import java.io.IOException
import java.io.InputStream
import java.io.InputStreamReader

object AssertsFileHelper {
    fun readFileFromAsserts(instrumentation: Instrumentation, file: String): String {
        val stream = instrumentation.context.resources.assets.open(file)

        val sb = StringBuilder()

        var strLine: String?
        val reader = BufferedReader(InputStreamReader(stream))

        strLine = reader.readLine()
        while (strLine != null) {
            sb.append(strLine)
            strLine = reader.readLine()
        }

        return sb.toString()
    }

}
